artemis-commons/src/main/java/org/apache/activemq/artemis/utils/collections/PriorityLinkedListImpl.java

```diff
@@ -20,18 +20,17 @@
 import java.util.Comparator;
 import java.util.NoSuchElementException;
 import java.util.concurrent.atomic.AtomicIntegerFieldUpdater;
-import java.util.function.ToLongFunction;
 
 /**
  * A priority linked list implementation
  * <p>
  * It implements this by maintaining an individual LinkedBlockingDeque for each priority level.
  */
-public class PriorityLinkedListImpl<T> implements PriorityLinkedList<T> {
+public class PriorityLinkedListImpl<E> implements PriorityLinkedList<E> {
```

Comment:
sorry, I have a massive number of changes.. and I thought one more fix wouldn't bother :)

Reply:
I tend to go the other way...if it is already a massive change, including unnecessary or unrelated changes just makes it worse and so I tend to avoid it at all costs. It makes reviewing the massive change harder, and then the 'other change' also tends to gets lost in the noise of the much larger main one. If its something that later turns out to be important (obviously this shouldn't though) its then also becomes much harder to look into later if needed due to the massive change in the way.

Reply:
this is a separate commit now. And there's a separate PR sending the same commit at #3641